plugins/admission/go.mod

```diff
@@ -7,15 +7,16 @@ replace github.com/fairwindsops/insights-plugins/plugins/admission => ./
 replace github.com/fairwindsops/insights-plugins/plugins/opa => ../opa
 
 require (
-	// IMPORTANT: Please also update the const  constant in pkg/pluto/pluto.go
```

Comment:
Unfortunately these got removed, I imagine by a command that automatically edits go.mod like `go mod tidy`?

Reply:
Oops good catch